packages/posix-base/posix-base.2.0.0/opam

```diff
@@ -9,7 +9,7 @@ bug-reports: "https://github.com/savonet/ocaml-posix/issues"
 depends: [
   "dune" {> "2.5"}
   "integers"
-  "ctypes"
+  "ctypes" {>= "0.6.0"}
```

Comment:
```suggestion
  "ctypes" {>= "0.12.1"}
```
This is when signed/unsigned were removed from the public interface. Currently that creates an interface conflict:
```
# Error: Files /home/opam/.opam/4.06/lib/integers/integers.cmxa
#        and /home/opam/.opam/4.06/lib/ctypes/ctypes.cmxa
#        both define a module named Unsigned
```